gnovm/pkg/gnomod/pkg.go

```diff
@@ -111,17 +114,15 @@ func ListPkgs(root string) (PkgList, error) {
 			return fmt.Errorf("validate: %w", err)
 		}
 
+		imports, err := gnoimports.PackageImports(path)
+		_ = err // get valid imports while ignoring bad/partial files
```

Comment:
Why not just do `imports, _ := ...`?

Reply:
I thought golangci-lint would yell at me for an unchecked error but turns out our config doesn't care
Also this makes it extra explicit that we ignore the error intentionally